R/pkg/R/DataFrame.R

```diff
@@ -3769,3 +3769,33 @@ setMethod("alias",
             sdf <- callJMethod(object@sdf, "alias", data)
             dataFrame(sdf)
           })
+
+
+#' broadcast
+#' 
+#' Return a new SparkDataFrame marked as small enough for use in broadcast joins. 
+#' 
+#' Equivalent to hint(x, "broadcast).
```

Comment:
I double check this but for some reason `\code` here made `roxygen` unhappy when I tried it last time.